taxcalc/functions.py

```diff
@@ -478,11 +485,9 @@ def StdDed(DSI, _earned, STD, e04470, e00100, e60000,
     c04100 = c04100 + e15360
 
     # ??
+    x04500 = 0.
     if f6251 == 0 and e04470 == 0:
         x04500 = e00100 - e60000
-        c04500 = c00100 - x04500
```

Comment:
Sean,

Are you testing all these patches to see if they do in fact improve the 
result? I am not able to keep up with all of them and analyze fully what I 
think. It would help if you took the time to write a few sentences about 
why the change is being made. This isn't twitter - you can write more tha 
140 characters.

On Fri, 2 Oct 2015, Sean.Wang wrote:

> 
> In taxcalc/functions.py:
> 
> >      if f6251 == 0 and e04470 == 0:
> >          x04500 = e00100 - e60000
> > -        c04500 = c00100 - x04500
> 
> Not sure what this part is for.

I am trying to work on this part of the code this week and don't want to 
go into the current problems till I have working code.

dan

> 
> —
> Reply to this email directly or view it on
> GitHub.[AHvQVUoaE8ENVrsfe7M6m_sL5_u86sgKks5o3rQlgaJpZM4GHeOp.gif]
> 
> 
>